### ISSUE TITLE
Decorative animated gifs need pause/stop/close button for a11y

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1057]
Maybe consider rendering still images instead if "user prefers reduced motion" enabled

-- 

"US03 Reading the What Is Health Equity > What Is Health Equity? page
1.10"		N/A	Animations under "how do i join the movement" are not stoppable or safe_mWeb(iOS)_1.10	"Environment:
DEVICE: iPad Air 2
iOS Version: 14.5.1
Model Number: MH0W2LL/A
Chrome browser: 87.0.4280.163

Steps to reproduce:
1. Go to https://dev.healthequitytracker.org, Login with valid credentials
2. Navigate to the What Is Health Equity? page /whatishealthequity
3. Scroll down to the ""how do i join the movement"" section

Actual Results:
The 4 animations on the page appear to last longer than 3 sec. There is no way to safely stop the animation.

Expected Result:
Animations should have a pause or last less than 3 sec

Video:
https://drive.google.com/file/d/1ukRN4dwRVuoxpChygmiqhq0xJUWxL9-z/view?usp=sharing&resourcekey=0-0tpS19LUW-HN1SKMd0Dhug"

**Comment by benhammondmusic** [comment]
https://www.joshwcomeau.com/react/prefers-reduced-motion/#the-hook

https://dev.to/eevajonnapanula/hiding-the-movement-gifs-play-buttons-and-prefers-reduced-motion-3499

**Comment by benhammondmusic** [comment]
https://css-tricks.com/gifs-and-prefers-reduced-motion/

**Comment by benhammondmusic** [comment]
#1056 adds support for "user prefers reduced motion".

For proper a11y support any animations lasting over 3 seconds should include a stop or pause (or close since these images are decorative) button.

**Comment by jgonzalezmsm** [comment]
i know those animated gifs are nice but we should probably talk about what that adds in terms of value and if there's a better use of screen real estate. if for short term it's trivial to add a pause button then i would say let's go ahead and do that for now

**Comment by benhammondmusic** [comment]
I agree. They're also used on several pages; I think we could certainly remove some of the occurrences. Pause button won't be too difficult but not priority for now. 

I did implement them automatically being Still images if a user requests reduced motion, so that helps for many affected users already.

**Comment by benhammondmusic** [comment]
Best practice may be replacing gifs with mp4s; it could significantly reduce file size / load time and provides native video play/pause features
